FirebaseCore.podspec

```diff
@@ -34,6 +34,6 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
   s.dependency 'GoogleToolboxForMac/NSData+zlib', '~> 2.1'
   s.pod_target_xcconfig = {
     'OTHER_CFLAGS' => '-fno-autolink ' +
-      '-DFIRCore_VERSION=' + s.version.to_s + ' -DFirebase_VERSION=4.8.0'
+      '-DFIRCore_VERSION=' + s.version.to_s + ' -DFirebase_VERSION=5.0.0'
```

Comment:
Have we been missing this in the updates?

Reply:
Yes :(  Luckily, not impacting official release so far.

Also just pushed another update so we tag components specifically.